gpcontrib/gg_tables_tracking/sql/gg_tables_tracking_guc.sql

```diff
@@ -0,0 +1,226 @@
+-- start_ignore
+\! gpconfig -c shared_preload_libraries -v 'gg_tables_tracking'
```

Comment:
Don't touch other extensions in `shared_preload_libraries`!
For example
```suggestion
gpconfig -c shared_preload_libraries -v "$(psql -At -c "SELECT array_to_string(array_append(string_to_array(current_setting('shared_preload_libraries'), ','), 'gg_tables_tracking'), ',')" postgres)"

```